src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_link">
+          <li class="nav_item">
+            <a
+              class="nav_list is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Laptops & Computers"
+              data-qa="hover"
+            >
```

Comment:
HTML Content issue: Don't use spaces in `<a>` tag's `href` property. Anchor links starts with the `#` symbol. In your case, it should be `#LaptopsAndComputers` instead of `#Laptops & Computers`.